content/docs/security/ratelimit/global.md

```diff
@@ -0,0 +1,436 @@
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.kgateway.dev/v1alpha1
+kind: TrafficPolicy
+metadata:
+  name: request-path
+  namespace: {{< reuse "docs/snippets/namespace.md" >}}
+spec:
+  targetRefs:
+  - group: gateway.networking.k8s.io
+    kind: Gateway
+    name: http
+  rateLimit:
+    global:
+      descriptors:
+      - entries:
+        - type: Path
+      extensionRef:
+        name: global-ratelimit
+EOF
+```
+
+{{% /tab %}}
+
+{{% tab tabName="User ID" %}}
+
+Limit requests based on a user ID from an `X-User-ID` header.
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.kgateway.dev/v1alpha1
+kind: TrafficPolicy
+metadata:
+  name: user-rate-limit
+  namespace: {{< reuse "docs/snippets/namespace.md" >}}
+spec:
+  targetRefs:
+  - group: gateway.networking.k8s.io
+    kind: Gateway
+    name: http
+  rateLimit:
+    global:
+      descriptors:
+      - entries:
+        - type: Header
+          header: "X-User-ID"
+      extensionRef:
+        name: global-ratelimit
+EOF
+```
+
+{{% /tab %}}
+
+{{% tab tabName="Combined globalrate limits" %}}
+
+Apply different limits based on both path and user ID. The following example rate limits requests along the path defined in the Rate Limit Service actions (1 per minute for `/path1` and 2 per minute for `/path2`) and requests with a `X-User-ID` header of `user1` (1 per minute).
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.kgateway.dev/v1alpha1
+kind: TrafficPolicy
+metadata:
+  name: combined-rate-limit
+  namespace: {{< reuse "docs/snippets/namespace.md" >}}
+spec:
+  targetRefs:
+  - group: gateway.networking.k8s.io
+    kind: Gateway
+    name: http
+  rateLimit:
+    global:
+      descriptors:
+      - entries:
+        - type: Path
+        - type: Header
+          header: "X-User-ID"
+      extensionRef:
+        name: global-ratelimit
+EOF
+```
+
+{{% /tab %}}
+
+{{% tab tabName="Local and global rate limits" %}}
+
+Combine local and global rate limiting in the same TrafficPolicy such as in the following example. For more information, see [Local rate limiting](../local).
+
+* Local: A rate limit of 5 requests per second is applied to all requests.
+* Global: A rate limit of 2 requests per minute is applied to requests with a `service` descriptor key of `premium-api`.
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.kgateway.dev/v1alpha1
+kind: TrafficPolicy
+metadata:
+  name: local-global-rate-limit
+  namespace: {{< reuse "docs/snippets/namespace.md" >}}
+spec:
+  targetRefs:
+  - group: gateway.networking.k8s.io
+    kind: Gateway
+    name: http
+  rateLimit:
+    local:
+      tokenBucket:
+        maxTokens: 5
+        tokensPerFill: 1
+        fillInterval: "1s"
+    global:
+      descriptors:
+      - entries:
+        - type: Generic
+          generic:
+            key: "service"
+            value: "premium-api"
```

Comment:
curious, does the generic descriptor need to have a key that is defined in the rate limit server config? I am assuming yes